<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgb(149, 221, 255), stop:1 rgba(255, 255, 255, 192));
	border-radius:15px;
}
QLineEdit{
	background-color:rgba(255, 193, 173, 192);
	border-radius:25px;
	padding: 5px;
 	font-size: 15px;
}
QToolButton{
border-style:none;
}
QToolButton#confirmBtn{
color:rgb(255,255,255);
background-color: rgb(253, 19, 68);
border-radius:25px;
}
QToolButton#confirmBtn::disabled,QToolButton#confirmBtn::pressed{
color:rgb(255, 220, 226);
background-color:rgb(255, 166, 186);
}
QToolButton#closeButton::hover{
color:rgb(255,255,255);
background-color: rgba(255, 56, 56, 150);
border-top-right-radius:15px;
}
QToolButton#minimizeButton::hover{
border-style:none;
background-color: rgba(203, 203, 203, 150);
}
</string>
   </property>
   <widget class="QToolButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>╳</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>一</string>
    </property>
   </widget>
   <widget class="QLabel" name="topicLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#e39802;&quot;&gt;注册账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="accountEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>为您账户设置昵称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="re_pswEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>请再次输入密码</string>
    </property>
   </widget>
   <widget class="QToolButton" name="confirmBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>确 认</string>
    </property>
   </widget>
   <widget class="QToolButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请设置一个密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
